easybuild/easyconfigs/m/MONAI/MONAI-0.8.0-foss-2021a.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonPackage'
+
+name = 'MONAI'
+version = '0.8.0'
+
+homepage = 'https://monai.io/'
+description = """
+MONAI is a PyTorch-based, open-source framework for deep learning in healthcare
+imaging, part of PyTorch Ecosystem.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+github_account = 'Project-MONAI'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = [('b427c117250b24199ea88c0b3ec1a6d80a22958b4413b7c85c6318ef011f6479',
+              'a63df7d5a680d9641c223ea090ff843a7d6f20bdb62095bd44f3b0480a4706ed')]
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
```

Comment:
why are you adding `CUDA` to the non-CUDA version? I would remove it unless really needed.

Reply:
That was an oversight - Thank you, consider it removed!